variables.tf

```diff
@@ -292,3 +292,9 @@ variable "instance_cbr_rules" {
   default     = []
   # Validation happens in the rule module
 }
+
+variable "skip_iam_authorization_policy" {
+  type        = bool
+  description = "Set to true to skip the creation of IAM authorization policy"
```

Comment:
Can you add more details on the auth policy that is created so the consumers knows what they are opting out of